Source/SwiftLintBuiltInRules/Rules/Idiomatic/NoMagicNumbersRule.swift

```diff
@@ -241,6 +254,19 @@ private extension ExprSyntaxProtocol {
         }
         return false
     }
+
+    func isContainedInExcludedMacro(_ macrosToIgnore: Set<String>) -> Bool {
```

Comment:
This could be done in an implementation of `visit(node: MacroExpansionExprSyntax)` that returns `.skipChildren` if the name matches.